src/app/main/component/user/components/profile/edit-profile/edit-profile.component.ts

```diff
@@ -59,6 +59,23 @@ export class EditProfileComponent extends FormBaseComponent implements OnInit, O
     }
   };
 
+  emailPreferencesList = [
+    { controlName: 'system', translationKey: 'system', periodicityControl: 'periodicitySystem' },
+    { controlName: 'likes', translationKey: 'likes', periodicityControl: 'periodicityLikes' },
+    { controlName: 'comments', translationKey: 'comments', periodicityControl: 'periodicityComments' },
+    { controlName: 'invites', translationKey: 'invites', periodicityControl: 'periodicityInvites' },
+    { controlName: 'places', translationKey: 'places', periodicityControl: 'periodicityPlaces' }
+  ];
+
+  periodicityOptions = [
```

Comment:
Using an enum or constants  is a great way to improve maintainability, type safety, and clarity in your code
you can put into some file like as `models/edit-profile-const.ts`